src/rviz/default_plugin/effort_display.h

```diff
@@ -574,6 +575,15 @@ class MessageFilterJointStateDisplay: public _RosTopicDisplay
       context_->queueRender();
     }
 
+  virtual void updateQueueSize()
+    {
+      tf_filter_->setQueueSize( (uint32_t) queue_size_property_->getInt() );
+      unsubscribe();
```

Comment:
did you add the `unsubscribe` and `resubscribe` calls because you found that they were necessary? most places (e.g. [marker display](https://github.com/ros-visualization/rviz/blob/7970ba08cee3810cfa1609c3b0f5136970eb2f7c/src/rviz/default_plugin/marker_display.cpp#L143)) just call `setQueueSize` and nothing else, but that might be what's beneath issue https://github.com/ros-visualization/rviz/issues/1135.